ot/lowrank.py

```diff
@@ -0,0 +1,289 @@
+"""
+Low rank OT solvers
+"""
+
+# Author: Laurène David <laurene.david@ip-paris.fr>
+#
+# License: MIT License
+
+
+import warnings
+from .utils import unif, get_lowrank_lazytensor
+from .backend import get_backend
+
+
+def compute_lr_sqeuclidean_matrix(X_s, X_t, nx=None):
+    """
+    Compute low rank decomposition of a sqeuclidean cost matrix.
```

Comment:
squared euclidean distance matrix* in the documentation. Plus could you specify the parameters as in `lowrank_sinkhorn` ?